_config.yml

```diff
@@ -36,6 +36,10 @@ plugins:
 
 medias:
   items:
+    - title: "Ils parlent de nous"
```

Comment:
```suggestion
    - title: "Intervention AI for good"
```

Reply:
Sorry was not specify in the issue, could you change the title to Intervention AI for good ? thanks